.gitignore

```diff
@@ -5,3 +5,4 @@
 /var/
 *.idea
 *~
+/Documentation-GENERATED-temp
```

Comment:
The lines in this file are sorted. Could you please move the new line to the corresponding place to retain the sorting?